pyproject.toml

```diff
@@ -18,6 +18,7 @@ psycopg2 = "^2.8.5"
 postgres = "^3.0.0"
 python-decouple = "^3.3"
 uvicorn = "^0.11.5"
+geopandas = "^0.7.0"
```

Comment:
Can you rather add this to dev-dependencies? The API doesn't need this, so I'd rather we just require it on our dev machines.